docs/modules/operation/pages/device-config-backup/dcb-requisition.adoc

```diff
@@ -22,4 +22,6 @@ If you want to enable device config backup on existing nodes in a requisition, s
 .. `schedule`: a cron expression to specify the time to fetch the device backup configuration (default is `0 0 0 * * ?`, every 24 hrs at 12:00 a.m.)
 . Click *Save* to save the metadata and then *Save* to save the requisition.
 . Click *Return* and click *Synchronize*.
-. Choose a scan option and click *Synchronize*.
\ No newline at end of file
+. Choose a scan option and click *Synchronize*.
+
+NOTE: By specifying the value `never` for the key `schedule` device configuration backups will not by triggered automatically. Of course it is still possible to retrieve backups on demand by using the Web-UI or the ReST services.
```

Comment:
NOTE: By specifying the value `never` for the key `schedule`, device configuration backups will not by triggered automatically. It is still possible to retrieve backups on demand by using the Web UI or the REST services.

Reply:
done.